projects/pre-sentence-reports-to-delius/src/main/resources/application.yml

```diff
@@ -0,0 +1,94 @@
+# Default config
+spring:
+  jpa:
+    database-platform: org.hibernate.dialect.Oracle12cDialect
+    hibernate.ddl-auto: validate
+  jms:
+    template.default-destination: pre-sentence-report-events
+    listener:
+      concurrency: 1
+      max-concurrency: 10
+      acknowledge-mode: client
+  security.oauth2.client:
+    registration:
+      pre-sentence-reports:
+        provider: hmpps-auth
+        authorization-grant-type: client_credentials
+        client-id: ${pre-sentence-reports-to-delius.client-id}
+        client-secret: ${pre-sentence-reports-to-delius.client-secret}
+    provider:
+      hmpps-auth:
+        token-uri: http://localhost:${wiremock.port}/auth/oauth/token
+
+delius.db.username: pre-sentence-reports-to-delius
+
+management.endpoints.web:
+  base-path: /
+  exposure.include: "health"
+
+feign.client.config:
+  default:
+    logger-level: full
+    connect-timeout: 5000
+    read-timeout: 5000
+    default-request-headers:
+      Accept: application/json
+      Content-Type: application/json
+
+
+---
+# Shared dev/test config
+spring.config.activate.on-profile: [ "dev", "integration-test", "oracle" ]
+
+context:
+  initializer:
+    classes: uk.gov.justice.digital.hmpps.wiremock.WireMockInitialiser
+
+spring:
+  activemq.broker-url: vm://localhost?broker.persistent=false&broker.useShutdownHook=false
+
+  datasource.url: jdbc:h2:file:./dev;MODE=Oracle;DEFAULT_NULL_ORDERING=HIGH;AUTO_SERVER=true;AUTO_SERVER_PORT=9092
+  jpa.hibernate.ddl-auto: create-drop
+
+wiremock:
+  enabled: true
+
+integrations:
+  pre-sentence-reports:
+    client-id: pre-sentence-reports-to-delius
+    client-secret: pre-sentence-reports-to-delius
+  alfresco:
+    url: http://localhost:${wiremock.port}
```

Comment:
We'll need to configure this in the values yml files.  I _think_ we'll already have network access.